reporter/html.js

```diff
@@ -707,8 +709,22 @@ QUnit.testDone(function( details ) {
 		}
 	}
 
+	localCollapse = config.collapse;
+
+	if (localCollapse !== collapseCheck) {
+		collapse = localCollapse;
+	}
```

Comment:
This can be simplified. Don't make a copy of `config.collapse`, instead always reference that directly. Then track if the first test was collapsed with a simple boolean flag, eg. `collapsedFirst`.